cmd/skywire-visor/commands/root.go

```diff
@@ -299,7 +215,8 @@ var rootCmd = &cobra.Command{
 
 func runVisor(conf *visorconfig.V1) {
 	var ok bool
-	log := initLogger(tag, syslogAddr)
+	mLog := initLogger(tag, syslogAddr)
```

Comment:
Why are we initializing this and then assinging it to `log`?

Reply:
apparently a mistake, I've changed it without issue